test/parallel/test-net-server-max-connections-close-makes-more-available.js

```diff
@@ -28,6 +29,11 @@ var createConnection = function(index) {
       sent.push(msg);
     });
 
+    connection.on('error', function(err) {
+      failed.push(index);
+      resolve();
+    });
+
```

Comment:
Perhaps add a check to make sure the `err.code` is ECONNRESET just to make sure some other problem doesn't mask an issue here?